hackathon_site/dashboard/frontend/src/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -2,12 +2,21 @@ import React from "react";
 import Typography from "@material-ui/core/Typography";
 import Header from "components/general/Header/Header";
 import { hackathonName } from "constants.js";
+import styles from "./AdminDashboard.module.scss";
+import { OrderCard } from "components/dashboard/OrderCard/OrderCard";
+import { OverviewCard } from "components/dashboard/OverviewCard/OverviewCard";
+import { InfoChart } from "components/dashboard/InfoChart/InfoChart";
 
 const AdminDashboard = () => {
     return (
         <>
             <Header />
-            <Typography variant="h1">{hackathonName} Admin Dashboard</Typography>
+            <div className={styles.dashboard}>
+                <Typography variant="h1">{hackathonName} Hardware Dashboard</Typography>
+                <OverviewCard />
```

Comment:
You don't need a new component for every single card. If you look at the design they are all the same with just text and icon differences, there's no need for a component for each but you can create a component for that row of cards to keep them together